changes/3010.feature.md

```diff
@@ -0,0 +1 @@
+Hide FastTrack(`pipeline`) menu by default on installation by script.
```

Comment:
1. I recommend avoiding the use of the technical term "pipeline." Users are primarily concerned with the disappearance of the "FastTrack" menu, and "pipeline" is an internal term that may confuse them.
2. Should we focus specifically on the scope of "installation by script"? It might help to clarify our objectives.

Additionally, we may want to consider renaming the menu_blocklist option to menu_hidelist or something similar in the next PR for better clarity.

Reply:
Agree. But let's leave it `pipeline` menu as is since the actual menu is regarded as `pipeline` for now.